src/style.css

```diff
@@ -0,0 +1,64 @@
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  padding: 22.5px 0;
```

Comment:
```suggestion
```
You can center the text vertically by using `display: flex` and `align-items: center`